Mesh/src/measurement.h

```diff
@@ -0,0 +1,142 @@
+#include "driver/gpio.h"
+#include "driver/pcnt.h"
+#include "ClosedCube_HDC1080.h"
+#include "esp_wifi.h"
+#include <driver/adc.h>
+#include "sensor_pinouts.h"
+#include <Wire.h>
+#include <driver/adc.h>
+#include <EEPROM.h>
+// #include "pinouts_and_constants.h"
+
+ClosedCube_HDC1080 hdc1080;
+
+int measurement_memory_start = 8; // first 8 bytes are reserved
+
+class Measurement
+{
+public:
+    unsigned int moisture_percent;
+    double temperature;
+    double humidity;
+    unsigned int light_level;
+    unsigned int battery_level;
+    void printMeasurement()
+    {
+        Serial.println("Moisture: " + String(moisture_percent) + "%");
+        Serial.println("Temperature: " + String(temperature) + "F");
+        Serial.println("Humidity: " + String(humidity) + "%");
+        Serial.println("Light Level: " + String(light_level));
+        Serial.println("Battery Level: " + String(battery_level) + "mV");
+    }
+};
+
+void measureSetup()
+{
+    digitalWrite(GPIO_NUM_26, HIGH);
```

Comment:
Just looking into clean code. Maybe we change this and other one liners that arent apparent what they do into functions, so this could be ENABLE_ACC_RAIL()